src/otx/algo/classification/vit.py

```diff
@@ -140,14 +139,18 @@ def _forward_explain_image_classifier(
             scores = pred_results.unbind(0)
             labels = logits.argmax(-1, keepdim=True).unbind(0)
 
-        return {
+        outputs = {
             "logits": logits,
             "feature_vector": feature_vector,
             "saliency_map": saliency_map,
-            "scores": scores,
-            "labels": labels,
         }
 
+        if not torch._C._is_tracing():  # noqa: SLF001
```

Comment:
```suggestion
        if not torch.jit.is_tracing():  # noqa: SLF001
```
Is it ok to use as above? If it's, how about avoiding to use internal funciton?

Reply:
Updated